controllers/managedocs_controller.go

```diff
@@ -1054,6 +1054,7 @@ func (r *ManagedOCSReconciler) reconcilePrometheus() error {
 
 		r.prometheus.Spec = desired.Spec
 		r.prometheus.Spec.ExternalLabels["clusterId"] = string(clusterVersion.Spec.ClusterID)
+		r.prometheus.Spec.ExternalLabels["addonName"] = r.AddonConfigMapName
```

Comment:
Where are we using addonName externalLabel?

Reply:
It will be useful in visualising all the data that comes from individual clusters

Reply:
Ack, so we will write a query to filter the metrics based on clusterID and addonName, right?

Reply:
Yup, something like this, to get the list of provider and consumer addons running
```
count by(addonName) (subscription_sync_total{addonName=~"ocs-(provider|consumer)", installed=~"ocs-osd-deployer.*"})
```

Reply:
Great! But IIRC we cannot push the metrics from consumer clusters to RHOBS, right?

Reply:
Yes, currently not. Still working on that

Reply:
Ack, thanks for clarification